app/component/map/map.cjsx

```diff
@@ -38,9 +38,6 @@ class Map extends React.Component
     if isBrowser
       origin = @context.getStore('EndpointStore').getOrigin()
 
-      if origin?.lat
-        fromMarker = <LocationMarker position={origin} className="from"/>
```

Comment:
Hides marker on front page, where it should be visible, possibly same on eg. stop, route and trip pages.